### PR TITLE
Implementing Multiple SSH Keys with Enhanced Management Features

### PR DESCRIPTION
**Pull request by Mahmoud-Emad**
### Description

Implemented multiple SSH key feature and SSH key management 

### Changes 
- Implemented a helper function to generate a name for the generated key to be saved with the key.
- Added activation and deletion logic for SSH keys.
- Worked on the feature to delete all selected keys, preparing to delete keys based on types like active or all keys.
- Unified UI elements by removing the tonal theme from cards to maintain consistency with the contracts list view.
- Enhanced UI by adding another dialog for importing SSH keys and updated the colors of the table.
- Calculated SSH key fingerprints, consolidated form dialogs, and introduced a key details dialog while removing redundant dialog.
- Refined the copy input wrapper to conditionally display the copy icon based on field data presence and improved the functionality of the generate dialog.
- Migrated old SSH keys to the new SSH key type and set them as active keys.
- Fixed the issue when saving SSH keys by loading them from the chain instead. Work in progress on updating the chain when pressing on active/inactive or deleting keys.
- Enhanced the way of saving/removing the keys and improved the import key form by adding a rule to validate the entered SSH key.
- Merged SSH key management functionality into all applications.

### Related Issues

- https://github.com/threefoldtech/tfgrid-sdk-ts/issues/1882

### Screenshots

- image
- image
- image
- image
- image
- image
- image
- image
- image

### Checklist

- [ ] Tests included
- [ ] Build pass
- [ ] Documentation
- [ ] Code format and docstrings
- [ ] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

File: packages/playground/src/components/ssh_keys/ManageSshDeployemnt.vue
Comment (MohamedElmdary) [inline packages/playground/src/components/ssh_keys/ManageSshDeployemnt.vue:1]: why are we using v2 syntax?

Comment (Mahmoud-Emad) [reply packages/playground/src/components/ssh_keys/ManageSshDeployemnt.vue:1]: Would you like to use V3 syntax? i see it's more readable also it I noticed that it used in many different places

Comment (MohamedElmdary) [reply packages/playground/src/components/ssh_keys/ManageSshDeployemnt.vue:1]: can  you mention where it's used so we make issue for updating it? @Mahmoud-Emad

**Review by amiraabouhadid — COMMENTED** [review]
It says that this sshkey was created today. However, this sshkey was created long before today. 

image

**Comment by MohamedElmdary** [comment]
Deploy button need to be disable if no key is select

### How to do that?
basically, use our validator form api but manually
example for such solution can be found here https://github.com/threefoldtech/tfgrid-sdk-ts/blob/development/packages/playground/src/components/node_selector/TfSelectionDetails.vue#L144

**Comment by Mahmoud-Emad** [comment]
> It says that this sshkey was created today. However, this sshkey was created long before today.
> 
> image

i have keys created from 31/3

image

**Review by MohamedElmdary — APPROVED** [review]
Good job

**Review by amiraabouhadid — APPROVED** [review]

**Pull request merged**